BUILD

```diff
@@ -8,6 +8,7 @@ filegroup(
         "AUTHORS",
         "BUILD",
         "LICENSE",
+        "MODULE.bazel",
```

Comment:
@meteorcloudy Should I also add WORKSPACE and WORKSPACE.bzlmod to the release?

Reply:
They are not needed when your project is used as a dependency, so probably not.